cves/kernel/CVE-2020-11725.yml

```diff
@@ -55,7 +55,12 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  NOT A SERIOUS VULNERABILITY... The report states that a variable stored as 
```

Comment:
Good description very straight to the point.

Reply:
Thank you, sir